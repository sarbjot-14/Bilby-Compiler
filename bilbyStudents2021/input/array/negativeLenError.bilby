main{
	imm g := alloc [int](-3);
	print g $n;


}

