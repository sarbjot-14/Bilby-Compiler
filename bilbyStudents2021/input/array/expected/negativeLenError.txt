Runtime error: negative array length
